cmd/containerd-stress/main.go

```diff
@@ -235,16 +235,6 @@ func (w *worker) getID() string {
 	return fmt.Sprintf("%d-%d", w.id, w.count)
 }
 
-func (w *worker) cleanup(ctx context.Context, c containerd.Container) {
```

Comment:
@crosbymichael Should this this have been called?

Reply:
There seems to a package level `cleanup()` function after this, maybe that's what is used now?